.drone.yml

```diff
@@ -37,13 +37,26 @@ steps:
     commands:
       - make lint
 
-  - name: build
+  - name: e2e
     image: golang:1.13
     commands:
-      - make build
+      - make e2e-test
     volumes:
       - name: deps
         path: /go
+    environment:
+      MCLI_PUBLIC_API_KEY:
+        from_secret: mcli_public_api_key
+      MCLI_PRIVATE_API_KEY:
+        from_secret: mcli_private_api_key
+      MCLI_OPS_MANAGER_URL:
+        from_secret: mcli_ops_manager_url
```

Comment:
[q] How does it setup/get the ops manager instance?

Reply:
talked offline